packages/headless/doc-parser/use-cases/insight.ts

```diff
@@ -0,0 +1,117 @@
+import {ActionLoaderConfiguration} from '../src/headless-export-resolvers/action-loader-resolver';
+import {ControllerConfiguration} from '../src/headless-export-resolvers/controller-resolver';
+import {EngineConfiguration} from '../src/headless-export-resolvers/engine-resolver';
+
+const controllers: ControllerConfiguration[] = [
+  {
+    initializer: 'buildFacet',
+    samplePaths: {},
```

Comment:
Just an fyi, these paths are used to display code samples in the reference doc. They don't need to be available right away but it would be good to add them progressively,

Reply:
Yeah sounds good. As we flesh out the insight panel use case for with a developer angle we'll improve the documentation.